ambry-tools/src/integration-test/resources/mocktio-extensions/DelayedAutoRebalancerHelixConfig.json

```diff
@@ -0,0 +1,15 @@
+{
+  "idealStateConfigFields": {
+    "numReplicas": 2,
+    "stateModelDefRef": "OnlineOffline",
+    "rebalanceStrategy": "org.apache.helix.controller.rebalancer.strategy.CrushEdRebalanceStrategy",
+    "minActiveReplicas": 0,
+    "rebalancerClassName": "org.apache.helix.controller.rebalancer.DelayedAutoRebalancer",
+    "rebalanceDelayInMins": 20
+  },
+  "clusterConfigFields": {
+    "maxOfflineInstancesAllowed": 4,
+    "numOfflineInstancesForAutoExit": 2,
+    "allowAutoJoin": true
+  }
+}
```

Comment:
You can move this to `config` directory. For testing, you can just instantiate a `Config` object (as you defined HelixVcrPopulateTool) and write it into string via jackson's `writeValueAsString` method.

Reply:
done